packages/web/src/lib/api.ts

```diff
@@ -1,22 +1,9 @@
 import algosdk, { Algodv2, ALGORAND_MIN_TX_FEE, SuggestedParams } from "algosdk";
 
-import { ChainType, TxParams } from "../types";
-import { betanetURL, mainnetURL, testnetURL } from "./constants";
+import { TxParams } from "../types";
 
-export function algoexplorerAlgod(chain: string): algosdk.Algodv2 {
-	const mainnetAlgoExplorer = new Algodv2("", mainnetURL, "");
-	const testnetAlgoExplorer = new Algodv2("", testnetURL, "");
-	const betanetAlgoExplorer = new Algodv2("", betanetURL, "");
-	switch (chain) {
-		case ChainType.MainNet:
-			return mainnetAlgoExplorer;
-		case ChainType.TestNet:
-			return testnetAlgoExplorer;
-		case ChainType.BetaNet:
-			return betanetAlgoExplorer;
-		default:
-			throw new Error(`Unknown chain type: ${chain}`);
-	}
+export function algoexplorerAlgod(walletURL: any): algosdk.Algodv2 {
```

Comment:
Here we should update it to the newly created type instead of `any`

Reply:
Yes I have update this!